### PR TITLE
Added active-passive architecture example

### PR DESCRIPTION
**Pull request by WilliamLoy**
Active-passive architecture can be used for use cases that cannot experience the slightest downtime that can occur when executing the restoration of a cluster.

### CODE REVIEW

File: docs/pages/management/operations/backup-restore.mdx [hunk 1]
```diff
@@ -239,3 +239,33 @@ of storage backend.
 </TabItem>
 </Tabs>
 
+### Active-passive Multi-region architecture example
```
Comment (zmb3) [inline docs/pages/management/operations/backup-restore.mdx:242]: - I'm not sure the backup restore page is the right place for this (it's certainly not where I'd look)
- Do we really recommend active-passive for mission-critical use cases? How many customers do we have running such an architecture?

Comment (alexfornuto) [reply docs/pages/management/operations/backup-restore.mdx:242]: @zmb3 would you be willing and able to suggest where you _would_ look for this kind of content?

Comment (pschisa) [reply docs/pages/management/operations/backup-restore.mdx:242]: We do have active/passive users in the wild.  Generally, these are the same users interested in https://github.com/gravitational/teleport/issues/16793 and are looking to have the shortest possible downtime for regional failures as Teleport is mission critical

**Comment by pschisa** [comment]
Closing out as @pnrao1983  is working on a separate guide for this discussion and a PR will be opened soon to cover the active/passive guide

**Pull request closed without merging**